### PR TITLE
Build foundation for CSPI docs & onboarding

### PR DESCRIPTION
**Pull request by ascerra**

### CODE REVIEW

File: docs/.gitbook.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+root: ./docs/
```
Comment (ryankwilliams) [inline docs/.gitbook.yaml:1]: @ascerra Is the plan to continue using GitBook or are we looking to use [GitHub pages](https://pages.github.com)? Since these docs are hosted on GitHub, may be worth looking into using pages. Which then could allow for using GitHub Action for auto deploying changes.

Comment (ascerra) [reply docs/.gitbook.yaml:1]: This is a good suggestion, I'll look into getting this done with github pages, thanks for the comment

Comment (ascerra) [reply docs/.gitbook.yaml:1]: @ryankwilliams I'm wondering if their is a reason other than Github Actions integration that made you think of this?

The reason I'm looking for more info is that with the [gitbook project that I set up](https://app.gitbook.com/o/NmndQIGBm1dypcvmouSa/s/yLsxuQuE6unUAM9hRiK0/) it is capable of automatically syncing based on any changes to the target branch. So automatically deploying changes seem to be the same for both.

With that I've experienced more complexity when working with github pages rather then gitbook. Gitbook needs just two config files `.gitbook` and `summary.md` the latter creating the flow of the docs pretty simply.

Github pages is more html driven. I've been [playing around](https://github.com/CSPI-QE/cspi-qe.github.io/tree/main/docs) with Jekyll templates that are supported for github pages but these are bringing with them a lot more code that we are getting for free with gitbook. Also I'm having trouble with linking to sub pages with github pages, though it could be the template or markdown specification being relied on.

Long story short I'm leaning towards shutting down my efforts with github pages and just using gitbook but I wanted to present this level of info to you first to make sure I'm not making this decision in a vacuum.

Comment (ryankwilliams) [reply docs/.gitbook.yaml:1]: @ascerra I primarily presented the question about using GitHub pages as it would mean its a one stop shop for everything (e.g. GitHub = repositories, docs).

I am totally okay to keep with gitbook for building/deploying documentation if that is easier to work with over using Jekyll. Please feel free to stop on this path and continue on with gitbook, if that works best. :)

Comment (ascerra) [reply docs/.gitbook.yaml:1]: Excellent thanks for the feedback Ryan!

We'll still be using this github repo as the source of truth for the documentation with gitbook synchronizing each time new code is merged into main.

Comment (ryankwilliams) [reply docs/.gitbook.yaml:1]: Sounds great @ascerra!

**Comment by ascerra** [comment]
This PR is no longer going to be pursued we are breaking down these parts into more manageable chunks in future PRs.

I'll leave this PR up until it is no longer relevant.

**Comment by calebevans** [comment]
@ascerra should we close this PR?
